lib/ScinanDevice.js

```diff
@@ -8,6 +8,9 @@ class ScinanDevice extends Homey.Device {
   async onInit() {
     this.token = this.homey.settings.get('token');
     this.device = this.getData();
+    this.summary = {
```

Comment:
Hva er this.summary, er det noe nytt i SDKv3, ser du oppdaterer den, men den blir ikke brukt.

Reply:
Det jeg oppdaget når jeg fikset buggen som sa at 'interval' ikke eksisterte var at selv om jeg oppdaterte interval parameteret ble det ikke brukt av setUpdateInterval. Vet ikke hvorfor, men regner med at settings objektet ikke blir oppdatert får onSettings koden har kjørt ferdig.

Jeg kikket litt på andre hvordan de hadde løst det, og en løste det ved å bruke et eget parameter. Trenger ikke hete summary, men når det oppdateres kan det brukes med en gang.